content/projects/oop/bank-accounts/part-1/_index.md

```diff
@@ -133,4 +130,8 @@ Also, please make use of assertions within your code to make sure that errors ar
 - withdraw a negative amount
 - deposit a negative amount
 - withdraw more money than you have
-- set a negative interest rate
\ No newline at end of file
+- set a negative interest rate
+
+**NB: Javascript assertions**
```

Comment:
This looks like it is meant to be a heading. Is there a reason why it is just bold text? It looks like it is inside the Java section and that is not ideal.

Please always use headings as headings.